expression/expression.go

```diff
@@ -126,10 +138,34 @@ func (s Schema) FindColumn(astCol *ast.ColumnName) (*Column, error) {
 		if (dbName.L == "" || dbName.L == col.DBName.L) &&
 			(tblName.L == "" || tblName.L == col.TblName.L) &&
 			(colName.L == col.ColName.L) {
-			if idx != -1 {
-				return nil, errors.Errorf("Column '%s' is ambiguous", colName.L)
+			if idx == -1 {
```

Comment:
Please update the comment of this function. This function has nothing to do with replacing.
"FindColumn find  an expression.Column from schema for a ast.ColumnName. It compares the db/table/column names. If there are more than one result, it will raise ambiguous error."